README.md

```diff
@@ -48,58 +55,109 @@ libraryDependencies += "com.google.cloud" % "google-cloud-firestore" % "1.35.2"
 ```
 [//]: # ({x-version-update-end})
 
-Authentication
---------------
+## Authentication
 
-See the
-[Authentication](https://github.com/googleapis/google-cloud-java#authentication)
-section in the base directory's README.
+See the [Authentication][authentication] section in the base directory's README.
 
-About Google Cloud Firestore
-----------------------------
+## Getting Started
 
-Cloud Firestore is a fully-managed NoSQL document database for mobile, web, and server development from Firebase and Google Cloud Platform.  It's backed by a multi-region replicated database that ensures once data is committed, it's durable even in the face of unexpected disasters. Not only that, but despite being a distributed database, it's also strongly consistent and offers seamless integration with other Firebase and Google Cloud Platform products, including Google Cloud Functions.
+### Prerequisites
 
-Troubleshooting
----------------
+You will need a [Google Cloud Platform Console][developer-console] project with the Cloud Firestore [API enabled][enable-api].
 
-To get help, follow the instructions in the [shared Troubleshooting document](https://github.com/googleapis/google-cloud-common/blob/master/troubleshooting/readme.md#troubleshooting).
+[Follow these instructions][create-project] to get your project set up. You will also need to set up the local development environment by
+[installing the Google Cloud SDK][cloud-sdk] and running the following commands in command line:
+`gcloud auth login` and `gcloud config set project [YOUR PROJECT ID]`.
 
-Transport
----------
-Firestore uses gRPC for the transport layer.
```

Comment:
There's a `transport` field you can set to `grpc` in the `.repo-metadata.json` if you want to keep this section.

Reply:
Done